packages/terra-consumer-icon/scripts/src/generate-example/generateStaticIcons.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */
```

Comment:
yes and most of the code is in scripts folder which will move out in terra-toolkit in near future.  Terra-consumer-icon will have different sets of icon component and svg.